plugins/portals-list.js

```diff
@@ -332,7 +332,7 @@ window.plugin.portalslist.portalTable = function(sortBy, sortOrder, filter) {
 
   var length = window.plugin.portalslist.listPortals.length;
 
-  ["All", "Neutral", "Resistance", "Enlightened", "visited", "captured", "Scout Ctrl" ].forEach(function(label, i) {
+  ["All", "Neutral", "Resistance", "Enlightened", "Visited", "Captured", "Scout controlled" ].forEach(function(label, i) {
```

Comment:
Shouldn't the title ("Scout Controlled") be in all caps, as suggested by @perringaiden?